news/2018-07-20-the-qat-gazette-june-2018.md

```diff
@@ -0,0 +1,98 @@
+
+Stop by and give the [current draft](https://gist.github.com/Okorin/1a5f5d6f489494c26b8a4fec13ba69ba) a read, and drop your feedback and opinions in the [discussion thread](https://osu.ppy.sh/community/forums/topics/719936) to help shape the future and direction of osu!mania mapping!
+
+### New Quality Assurance Team additions
+
+This month, we welcome a new piano masher to the Quality Assurance Team. Formerly a long standing member of the Beatmap Nominators, [Maxus](https://osu.ppy.sh/users/4335785) will be assisting the Quality Assurance Helpers with checking and disqualifying beatmaps, and giving [Feerum](https://osu.ppy.sh/users/4815717) and myself a helping hand on the osu!mania community side of things. We're glad to have him on board!
+
+On the osu! side of the coin, another new team member has arisen: [Hobbes2](https://osu.ppy.sh/users/8157492). Hobbes' recent output of checked beatmaps in the Quality Assurance Helpers has been frankly astonishing, and we hope that trend continues in the QAT to keep on top of the ever-growing pool of osu! beatmaps making their way to the Qualified section.
+
+### Quality Assurance Team retirements
+While not a retirement exactly, the Quality Assurance Team underwent a bit of spring cleaning this month, where we evaluated the direction and focus we want to have as a team, specifically one that is geared mostly towards disqualifications and beatmap checking.
+
+As a result, [Doyak](https://osu.ppy.sh/users/2046893) and [Kurai](https://osu.ppy.sh/users/77089) have been moved over to the Global Moderation Team, which better aligns with their previously moderation-heavy duties within the QAT.
+
+Following a more traditional retirement, [Kurokami](https://osu.ppy.sh/users/260933) has left the team after a faithful 3 years of service, rightfully earning him the 3 Year Quality Assurance Team badge to adorn his profile page.
+
+His contributions to the Beatmap Spotlights and to the osu!catch community in particular have inspired many others to seize the initiative and further the projects they love. These efforts are greatly appreciated and will not be forgotten any time soon. Despite no longer being part of the team, he still continues to work and improve on Beatmap Spotlights, so continue to expect more from him in the future!
+
+As the team's focus shifts, [Asherz007](https://osu.ppy.sh/u/9014047) has also decided to take his leave from the team and return to the more traditional hands-on duties of a Beatmap Nominator. He can still be found modding and nominating the osu!mania maps we all love however, and is also helping to organize the osu!mania duties of the Mapping Mentorship Program, only with a purple name instead of a red one. There's sure to be more from him in the future as he shows no signs of slowing down.
```

Comment:
This user link is the only one with the `/u/`.

Reply:
✅